### PR TITLE
Add clarity on what Semgrep Assistant does in PR/MR vs Slack

### PR DESCRIPTION
**Pull request by armchairlinguist**
Clarify that Assistant doesn't supply autofixes in Slack.

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team

### CODE REVIEW

**Review by underyx — APPROVED** [review]

File: docs/semgrep-code/semgrep-assistant-code.md [hunk 1]
```diff
@@ -192,11 +192,11 @@ The following are recommendations users can receive from Semgrep Assistant.
 
 ### Analyze if a finding is a true or false positive
 
-Semgrep Assistant can analyze if your finding is a true or false positive. The accuracy of its recommendations is roughly 60% and varies based on the language and framework you are using. This is its default use case.
+Semgrep Assistant can analyze if your finding is a true or false positive. The accuracy of its recommendations is roughly 60% and varies based on the language and framework you are using. This is its default use case. This feature is active in both PR/MR comments and Slack notifications.
```
Comment (khorne3) [inline docs/semgrep-code/semgrep-assistant-code.md:195]:
```suggestion
Semgrep Assistant can analyze if your finding is a true or false positive. The accuracy of its recommendations is roughly 60% and varies based on the language and framework you are using. This is its default use case. This feature is active in both PR and MR comments and Slack notifications.
```

File: docs/semgrep-code/semgrep-assistant-code.md [hunk 2]
```diff
@@ -192,11 +192,11 @@ The following are recommendations users can receive from Semgrep Assistant.
...
 
 ### Suggest autofix code snippets to resolve the finding
 
-Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep rules which do not have human-written autofix suggestions.
+Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep rules which do not have human-written autofix suggestions. This feature is only active in PR/MR comments.
```
Comment (khorne3) [inline docs/semgrep-code/semgrep-assistant-code.md:199]:
```suggestion
Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep rules which do not have human-written autofix suggestions. This feature is only active in PR and MR comments.
```

**Review by khorne3 — APPROVED** [review]
Small nits, but LGTM otherwise. Thank you!

File: docs/semgrep-code/semgrep-assistant-code.md (hunk 1 above)
Comment (armchairlinguist) [reply docs/semgrep-code/semgrep-assistant-code.md:195]: @khorne3 Thanks, I wasn't sure how best to refer to the comments here - I know we don't love the PR/MR notation. This change does feel a tad weird to me because without any further punctuation, this can sound like "in [PR] and [MR comments] and [Slack notifications]". I don't think it's a super likely interpretation so I may just go ahead and commit the suggestion, but if you have thoughts on improvement I'd be happy to do a quick followup.

Comment (khorne3) [reply docs/semgrep-code/semgrep-assistant-code.md]: @armchairlinguist Yeah, I can see that -- I've taken to writing "pull request and merge request comments" which I understand is a bit wordy but a bit clearer (maybe? Ha.)
